### PR TITLE
Adds a new Cyborg sprite for all the basic departments. 

### PR DESCRIPTION
**Pull request by TravellingMerchant**
Adds brand new Cyborg sprites for public use. Lore wise, they are from a line of robots called Noble, with their department type after their name. For example: Noble-SEC or Noble-ENG.

Includes:
Standard
Janitorial
Security
Medical
Mining
Service

A Gif of all the versions can be found here. 
noble-1

The only difference is that the Engineering version has a flickering welder flame on its welding arm.

### CODE REVIEW

**Comment by Draconic-saint** [comment]
Did you get the sprites to fit within 32x32? I seemed to recall you had some issues with that.

**Comment by TravellingMerchant** [comment]
Yep, all normally sized in 32x32

**Comment by FlattestGuitar** [comment]
They're still too big, in my opinion, but damn they look awesome.

**Comment by davidchan13** [comment]
So lesser form lings and genetics who monkey themselves can break into the armory and go on killing sprees?

**Comment by Draconic-saint** [comment]
They look mech sized to me, but yeah, looking good nonetheless

**Comment by SamHPurp** [comment]
They look too much like mechs... 👎

**Comment by taukausanake** [comment]
Would it be possible to make a type of Noble upgrade for borgs into this? If people are saying they look like mechs then I'd go all in and make them minor mechs of sorts. Not completely like so, no piloting and not as good stats but better than normal borgs at least. I can just imagine larger robots serving and monitoring the crew. Doesn't fit with your lore though so not sure what to make there

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -289,6 +289,7 @@ var/list/robot_verbs_default = list(
 			module_sprites["Basic"] = "robot_old"
 			module_sprites["Android"] = "droid"
 			module_sprites["Default"] = "robot"
+			module_sprites["Noble-STD"] = "Noble-STD"
```
Comment (Fox-McCloud) [inline code/modules/mob/living/silicon/robot/robot.dm]: a noble STD

🤔

Comment (TravellingMerchant) [inline code/modules/mob/living/silicon/robot/robot.dm]: Hah, I can change it if you want me to, but that is the normal abbreviation of standard!

**Comment by KasparoVy** [comment]
Awesome to see these finished. Looks totally badass

**Comment by Fox-McCloud** [comment]
Finally a cyborg sprite that I would remotely consider using over default.

**Pull request merged**